app/views/reviews/index.html.haml

```diff
@@ -1,6 +1,11 @@
 .scroll-panel-header
   %h3.scroll-panel-title Recenzje
 .scroll-panel-body
+  = simple_search_form_for @query, @query_params do |f|
+    = f.input :status_start, label: "Status"
+    = f.select :status, options_from_collection_for_select(Review.all, "status", @search.status) 
```

Comment:
Ponieważ korzystamy z simple_form_form, to wystarczy wpisać:
`f.input :status_start, collection: Review::STATUS_MAPPING.keys, include_blank: true`